docs/source/inference.rst

```diff
@@ -0,0 +1,206 @@
+Inference Using Pretrained Embedding
+--------------------------------------
+
+Users can use DGL-KE to do inference tasks based on pretained embeddings (We recommand using DGL-KE to generate these embedding). Here we support two kinds of inference tasks:
+
+  * **Linkage score ranking** Given a list of (h, r, t) triplets, calculate the linkage score using the predefined score function for each triplet, sort the resulting scores and output the topk most confident triplets.
+  * **Embedding similarity ranking** Given a list of (e, e) enitity pairs or (r, r) relation pairs, calculate the similarity of for each pair, sort the resulting similarity score and output the topk most similar pairs.
```

Comment:
finding similar embeddings: given an entity/relation embedding, find the most similar entity/relation embeddings for some pre-defined similarity functions.